recipes/SimpleElastix/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "0.9.0" %}
+
+package:
+  name: example
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/kaspermarstal/SimpleElastix.git
+  git_tag: 93b1482100c9fa8fba156b4d8f28f0388debc38e
```

Comment:
We prefer released version, but we are OK with a specific `tag` as long as you have a reason. Can you write a comment to why you are using that `tag`?